api/src/services/mail/templates/user-registration.liquid

```diff
@@ -0,0 +1,36 @@
+{% layout 'base' %} {% block content %}
+
+<h1>Verify your email address</h1>
+
+<p style='padding-bottom: 30px'>
+  Thanks for registering with us at <i>{{ projectName }}</i>.
+  To complete your registration you need to verify your email address by opening the following verification-link.
+</p>
```

Comment:
Should we add one of those standard disclaimers a la "If you don’t want to create an account, you can ignore this message."

Reply:
Yeah, sounds good, agreed! :+1: